config/test.config.js

```diff
@@ -0,0 +1,6 @@
+var nconf = require('nconf');
+
+nconf.argv()
+  .env()
+  .file({ file: './config/test.config.json' });
```

Comment:
since this config is contained in the repo, use .defaults: [JSON] instead of creating a test config file